start_stop.lua

```diff
@@ -460,6 +442,8 @@ function courseplay:start(self)
 		self.cp.changedMRMod = true;
 	end
 	if self.cp.drivingMode:get() == DrivingModeSetting.DRIVING_MODE_AIDRIVER then
+		local ret_removeLegacyCollisionTriggers = false
```

Comment:
what's the purpose of this variable?

Reply:
See next line - it gets the return value from the function call.
In general I think it is a good approach to get a feedback from a function if it was successful or not.
This could be used for further processing / error handling / information to the user...
I just want to make the developer aware that there is a result provided from the function, not only fire&forget

Reply:
Then put a TODO comment in there. These are marked as unused variables in my IDE which does not look good :)

Reply:
OK